test/logjam/framework/timbre_test.clj

```diff
@@ -0,0 +1,74 @@
+(ns logjam.framework.timbre-test
+  (:require
+   [clojure.test :refer [deftest is testing use-fixtures]]
+   [logjam.framework :as framework]
+   [logjam.framework.timbre :as sut]
+   [taoensso.timbre :as timbre]))
+
+;; for `match?`
+(require 'matcher-combinators.test)
```

Comment:
That's the point of requiring this separately?

Reply:
> That's the point of requiring this separately?

So that no tools could accidentally remove "unused requires"